src/main/steps/common/content.utils.ts

```diff
@@ -63,3 +63,7 @@ export const getAddressFields = (addressPrefix: string, userCase: Partial<CaseWi
 export const accessibleDetailsSpan = (spanText: string, accessibleText: string): string => {
   return spanText + '</span><span class="govuk-visually-hidden"> &nbsp - ' + accessibleText;
 };
+
+export const accessibleProgressBarSpan = (progressBarLabel: string, isComplete: boolean): string => {
+  return progressBarLabel + `<span class="govuk-visually-hidden"> &nbsp; (${isComplete ? '' : 'not '}completed)`;
```

Comment:
why not do this in a template?

Reply:
@benouaer I actually had issues with adding it directly into the template, I didn't manage to identify why but it works through the content, and I made it into a common function so as to be more DRY with the code